src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In Javascript,if a string s='Intelligent' then what will be the value of s.slice(2,5)?",
+    Answer: "tell",
+    Distractor1: "tellig",
+    Distractor2: "ll",
+    Distractor3: "Inigent",
+    Explanation:
+      "The slice() method returns the extracted part in a new string.The slice() method does not change the original string.slice(strating index,ending index)",
+    Link: "https://www.w3schools.com/jsref/jsref_slice_string.asp"
```

Comment:
We only accept links from freeCodeCamp resources, official documentation or wikipedia.

```suggestion
    Link: "https://www.freecodecamp.org/news/javascript-substring-examples-slice-substr-and-substring-methods-in-js/"
```